setup_cfg_fmt.py

```diff
@@ -194,12 +189,12 @@ def _python_requires(
     for env in _tox_envlist(setup_cfg):
         match = TOX_ENV.match(env)
         if match:
-            version = _to_ver('.'.join(match.groups()))
+            version = _to_ver(f'3.{match.group(0)}')
```

Comment:
you want `match[1]` I believe

Reply:
Ah right I changed this before running tests. TIL you can use `__getitem__` on matches..

Reply:
yep!  it's new in 3.6 :D